### PR TITLE
[MM-43637] Change shortcut to open channel info RHS on mac

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
This PR changes the shortcut to CMD+SHIFT+I on mac only. Should still use CMD+ALT+I on Windows.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-43637

#### Related Pull Requests
N/A

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/sidebar_right/sidebar_right.jsx [hunk 1]
```diff
@@ -93,7 +96,7 @@ export default class SidebarRight extends React.PureComponent {
                 } else {
                     this.props.actions.openAtPrevious(this.previous);
                 }
-            } else if (e.altKey && Utils.isKeyPressed(e, Constants.KeyCodes.I)) {
+            } else if (Utils.isKeyPressed(e, Constants.KeyCodes.I) && (channelInfoShortcutMac || channelInfoShortcut)) {
```
Comment (julmondragon) [inline components/sidebar_right/sidebar_right.jsx:99]: nit: maybe a ternary here could make it more direct and executed only when needed.

**Review by julmondragon — APPROVED** [review]
Looks good!

**Review by pvev — APPROVED** [review]
LGTM! thanks @JulienTant

**Review by matthewbirtch — APPROVED** [review]

**Comment by stevemudie** [comment]
@JulienTant ⌘⇧i (**CMD+Shift+i**) worked as expected on macOS so that's all set.

However, nothing appears to work on Windows and it's worth mentioning that Windows PCs lack the CMD key so the PR description above that mentions "_Should still use CMD+ALT+I on Windows_" won't apply unless you meant the dedicated Windows key? 🤔

It's worth mentioning that alt+shift+i are configured on both Edge and Chrome on Windows to submit feedback so that's unavailable.
Windows Chrome
Windows Edge

**Comment by stevemudie** [comment]
> Oops I meant CTRL+ALT+I on Windows.

Unfortunately, it doesn't do anything. :/

**Comment by JulienTant** [comment]
So I tried using a VM and configured the shortcut and it seemed to work 🤔

https://user-images.githubusercontent.com/785518/166806969-db0ffe8b-344c-4f80-ab8e-24299b3c05bc.mp4

**Comment by matthewbirtch** [comment]
> So I tried using a VM and configured the shortcut and it seemed to work 🤔
> 
>  2022-05-04.11-59-57.mp4

It worked for me in a VM as well

**Review by stevemudie — APPROVED** [review]
I'm going to assume that it's my setup (Windows 11 on Parallels) that is the issue. Since it's working for @JulienTant and @matthewbirtch just fine I'll call it done.

It did work fine on macOS for me though.

**Pull request merged**